tests/CMakeLists.txt

```diff
@@ -53,6 +53,21 @@ set(tests
         test_sockopt_hwm
         test_heartbeats
         test_poller
+        test_atomics
+        test_bind_src_address
+        test_capabilities
+        test_ipc_wildcard
```

Comment:
We don't support IPC on windows yet, maybe test_ipc_wildcard should be part of the non windows tests?